composer.json

```diff
@@ -15,7 +15,8 @@
         "php": "^7.1",
         "symfony/framework-bundle": "^2.8|^3.0",
         "sonata-project/seo-bundle": "^2.2",
-        "doctrine/collections": "^1.0"
+        "doctrine/collections": "^1.0",
+        "symfony/phpunit-bridge": "^3.3"
```

Comment:
this should stay in require-dev. for installing this as a dependency, we must not depend on the phpunit-bridge